src/aqicnLayer.js

```diff
@@ -42,6 +42,12 @@ L.LayerGroup.AQICNLayer = L.LayerGroup.extend(
             self._map.spin(false);
           }
         });
+
```

Comment:
It would be a good idea to clear this timeout so that it doesn't keep running if the spinner disappears after a successful data fetch before ten seconds.